src/api/json/catalog.json

```diff
@@ -6160,6 +6160,12 @@
       "description": "A blogroll interchange format",
       "fileMatch": [],
       "url": "https://www.json-wf.org.uk/json-wf-schema-1.0.json"
+    },
+    {
+      "name": "Cloud Foundry Application Manifest",
+      "description": "A manifest describes a Cloud Foundry application and can be used to deploy it to a Foundation",
+      "fileMatch": ["manifest.yml", "manifest.yaml"],
```

Comment:
This is too generic a file match and will result in lots of false positive matches. Can you come up with a more specific name or include a more specific folder structure in the pattern?

Reply:
Hi @madskristensen, thanks for the review. Unfortunately I do not think I can. CF manifests are typically only named `manifest.yml` (there are examples on their site) and located in the root of the repository. 

Are there any ways to match schemas to documents besides filename/path?

Reply:
> Are there any ways to match schemas to documents besides filename/path?

That's the only way to do it automatically unless you've got an editor plugin/extension that can do it. You can also reference the schema URL directly in the manifest.yml file which is what we see commonly done in these situations.